readme.md

```diff
@@ -203,9 +203,23 @@ Points-free function definitions look just like normal assignments without `func
 
 TODO
 
-## Guarded Functions
+## Guards
+Guards are boolean expressions that prevent evaluation of code unless the guard evaluates to true.
+It can be simulated thanks to the conditional ternary Operator.
```

Comment:
I understand a guard as simply some boolean condition that prevents the guarded expression from being evaluated if that condition does not pass. A ternary or a simply if statement can be considered a guard in the right context, no? 

Chained `&&` and short circuiting also is a way to do that too in JS and many other languages, maybe it'd be a good idea to provide a few short examples of this?